File: 2019/1.py
print(sum([int(int(x[:-1])/3)-2 for x in open("1.txt").readlines()]))
print(sum([[f:=lambda x: int(x/3)-2+f(int(x/3)-2) if x > 5 else 0,f(int(x[:-1]))][-1] for x in open("1.txt").readlines()]))

File: 2022/1.jl
lines = readlines("1.txt")

maxsum = 0
sum = 0
println(getindex(lines, [1, 2, 3, 4, 5, 6]))
sums = Int64[]
for line in lines
    global sum, maxsum
    if line == ""
        maxsum = max(maxsum, sum)
        push!(sums, sum)
        sum = 0
    else
        sum += parse(Int64, line)
    end
end
println(maxsum)
sort!(sums)
n = length(sums)
println(sums[n], " ", sums[n-1], " ", sums[n-2], ", ", sums[n] + sums[n-1] + sums[n-2])